src/plugins/intel_gpu/src/plugin/ops/multiclass_nms.cpp

```diff
@@ -0,0 +1,86 @@
+// Copyright (C) 2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <intel_gpu/primitives/multiclass_nms.hpp>
+#include "ngraph_ops/multiclass_nms_ie_internal.hpp"
+
+#include "intel_gpu/plugin/common_utils.hpp"
+#include "intel_gpu/plugin/program.hpp"
+#include "intel_gpu/primitives/multiclass_nms.hpp"
+#include "intel_gpu/primitives/mutable_data.hpp"
+
+namespace ov {
+namespace intel_gpu {
+
+
+static void CreateMulticlassNmsIEInternalOp(Program& p, const std::shared_ptr<ngraph::op::internal::MulticlassNmsIEInternal>& op) {
+    validate_inputs_count(op, {2, 3});
+
+    if (op->get_output_size() != 3) {
```

Comment:
I thinks this should not be so, because even though the operation is defined with 3 outputs, an output nodes might be optimized out from ngraph if it is not used by other node.

Reply:
Done.